be/src/vec/sink/vtablet_sink.cpp

```diff
@@ -510,6 +515,8 @@ Status VNodeChannel::open_wait() {
             }
         }
     });
+    _parent->_running_channels_num++;
```

Comment:
This is wrong because when open wait finished, there is no pending rpc

Reply:
done